<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdvParameterList</class>
 <widget class="QWidget" name="AdvParameterList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;h2&gt;Full Parameter List&lt;/h2&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTableWidget" name="tableWidget"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="loadPushButton">
         <property name="text">
          <string>Load From File</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="savePushButton">
         <property name="text">
          <string>Save To File</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="compareButton">
         <property name="text">
          <string>Compare with File</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="refreshPushButton">
         <property name="text">
          <string>Refresh</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="writePushButton">
         <property name="text">
          <string>Write</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="paramProgressBar">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string comment="Download Progress"/>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="progressLabel">
         <property name="text">
          <string>0/?</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="downloadRemoteButton">
         <property name="text">
          <string>Download Param Files</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
